app/controllers/help_controller.rb

```diff
@@ -45,7 +45,7 @@ def edit
   end
 
   def index
-    @help = help_source.sorted.page(params[:page]).per(25)
+    @help = help_source.sorted
```

Comment:
I think it's worth paginating this. (The view will need an update too)  We don't have these in place yet, but in the future, there could be a few hundred help pages.